packages/combobox/src/LionCombobox.js

```diff
@@ -420,13 +420,30 @@ export class LionCombobox extends OverlayMixin(LionListbox) {
     }
   }
 
-  /* eslint-disable no-param-reassign, class-methods-use-this */
+  /**
+   * We need to extend the repropagation prevention conditions here.
+   * Usually form groups with single choice will not repropagate model-value-changed of an option upwards
+   * if this option itself is not the checked one. We want to prevent duplicates. However, for combobox
+   * it is reasonable that an option can become unchecked without another one becoming checked, because
+   * users can enter any text they want, whether it matches an option or not.
+   *
+   * Therefore, extend the condition to fail by checking if there is any elements checked. If so, then we
+   * should indeed not repropagate as normally. If there is no elements checked, this will be the only
+   * model-value-changed event that gets received, and we should repropagate it.
```

Comment:
Nice debugging 👍 
Should we allow for unselecting a single choice even on choicegroup level?

Reply:
Perhaps.. Maybe the radio group is special in that it doesn't allow for unselecting, and this check is radio group specific. Then again, this behaviour works for listbox select (like rich select). So.. I'm not sure, this will be clearer if we have more use cases for single choice form fields I think. Maybe leave it as it is (default behaviour that you can't unselect) for now?

Reply:
That would be fine by me. The platform doesn't allow unselecting via UI, although I think when you build groups with role=radio this is a possibility (the polymer radio-group supported this at least)